portal/gil/templates/gil/portal.html

```diff
@@ -65,7 +65,7 @@ <h4 class="icon-box__lowhead">{{ _("Complete Registration") }}</h4>
     </section>
     <br class="clearfix"/>
     <br/>
-    {{footerHTML("home--footer-container")}}
+    {{footerHTML("home--footer-container", user)}}
```

Comment:
keyword arguments can be better for functions with 2 or more args; they're self-documenting and each function call doesn't need to be updated if the function signature changes
```suggestion
    {{footerHTML(custom_class="home--footer-container", user=user)}}
```